### PR TITLE
Added main structure(index.html/style.css)

### PR DESCRIPTION
**Pull request by braincee**
**_In this pull request, the following requirements were implemented:_**

1. I added main structure(index.html/style.css)
2. I  added setup linters
3. I added a README file
4. I added `gitignore` file with ignored files
5. I added CSS tree validator plugin
6. I added inline style disable Hint
7. I made the Portfolio Website mobile responsive using CSS media queries.
8. I made lots of error checks using linter setup
9.  I changed the `hamburger` png file and used font-awesome.
10. I run all my linter setup errors on my local terminal
11. I added more information on my `README` file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   <header>
+       <nav class="navbar">
+           <a href="#">LOGO</a>
+           <ul class="nav-menu">
+               <li class="nav-item"><a href="#" class="nav-link">PORTFOLIO</a></li>
+               <li class="nav-item"><a href="#" class="nav-link">CONTACT</a></li>
+               <li class="nav-item"><a href="#" class="nav-link">ABOUT</a></li>
+               <li class="nav-item">WORK</li>
+               <li class="nav-item"><a href="#" class="nav-link"><img src="images/Union.png" alt="mail"></a></li>
+               <li><img src="images/hamburger.png" alt="hamburger-icon" class="hamburger"></li>
```
Comment (uche-inyama) [inline index.html]: Kindly wrap your hamburger button implementation, inside a `<nav>` tag. :slightly_smiling_face:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+
+Built With Languages
+Html
+CSS
+
+Author 
+Stephen Annor 
+
+Accounts 
+Github: https://github.com/braincee
```
Comment (uche-inyama) [inline README.md]: Kindly add the missing headers in your readme file. Use this [template for reference](https://github.com/microverseinc/readme-template/blob/master/README.md)

[**optional**] For headers that might be redundant in this project, kindly leave them out, for example, **Install and Run Test**.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
Hi @braincee  :wave: 

Good job so far! your project is coming along just fine.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] In your next commit, add a bit touch to the messages - more description, when writing them :slightly_smiling_face: 

_Check the comments under the review._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (braincee) [reply index.html:22]: Please Mr Henry, You just reviewed my code and said I should make changes in the hamburger nav tag. I have a screenshot of the issue. The hamburger is already in a nav tag. Please should I remove it and put it into another nav tag. 
Like this:  <nav>
  hamburger-icon
</nav>
Waiting patiently for reply, Thanks

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
...
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+
+👤 STEPHEN ANNOR
+
+GitHub: @braincee
+Twitter: @annor_0543
+LinkedIn: https://www.linkedin.com/in/kwesi-appiah-1387801a1/
```
Comment (prolajumokeoni) [inline README.md:24]: ## OPTIONAL

- [ ] Kindly use this formats to make your links readable and more professional, check [microverse templates](https://github.com/microverseinc/readme-template/blob/master/README.md), you can fork it so you can always use it 👍🏿 

`GitHub: [@githubhandle](https://github.com/githubhandle)
Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    <meta name="viewport" content="width=device-width initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/53a64647bf.js" crossorigin="anonymous"></script>
+    <title>Portfolio-website</title>
+ </head>
+
+ <body>
+   <header>
+       <nav class="navbar">
+           <a href="#">LOGO</a>
```
Comment (prolajumokeoni) [inline index.html:16]: - [ ] Kindly use the same font color used in the figma design for your logo, please check screenshot below  👍🏿

File: index.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+               <li class="nav-item"><a href="#" class="nav-link"><img src="images/Union.png" alt="mail"></a></li>
+               <li class="bars"><i class="fa-solid fa-bars"></i></li>
+           </ul>
+       </nav>
+   </header>
+   <main class="myself-1">
+    <div class="container-2">
+        <section class="myself-2">
+            <h2 class="name">Stephen Annor</h2>
+            <h3 class="tech">SOFTWARE DEVELOPER</h3>
```
Comment (prolajumokeoni) [inline index.html:30]: ## OPTIONAL

- [ ] Kindly use the same color for the h2 tag just like the Figma design, you can make this a h1 tag so it serves as the heading of your website 👍🏿 
- [ ] Kindly make the h3 tag be on one line
- [ ] Kindly add the background image for this section 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
AA  |  SD

File: index.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+ <body>
+   <header>
+       <nav class="navbar">
+           <a href="#">LOGO</a>         
+           <ul class="nav-menu">
+               <li class="nav-item"><a href="#" class="nav-link">PORTFOLIO</a></li>
+               <li class="nav-item"><a href="#" class="nav-link">CONTACT</a></li>
+               <li class="nav-item"><a href="#" class="nav-link">ABOUT</a></li>
+               <li class="nav-item"><a href="#" class="nav-link"><img src="images/Union.png" alt="mail"></a></li>
+               <li class="bars"><i class="fa-solid fa-bars"></i></li>
```
Comment (prolajumokeoni) [inline index.html:22]: ## OPTIONAL

- [ ] Kindly increase the size of the hamburger menu icon, please make it look like the Figma design check screenshot below 👍🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi  @braincee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider inline comments in the review


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @prolajumokeoni and @uche-inyama 

_**Note: Please I have implemented all the required and optional changes.**_

thank-you

**Pull request merged**